ktor-server/ktor-server-plugins/ktor-server-websockets/jvm/test/io/ktor/tests/websocket/WebSocketTest.kt

```diff
@@ -118,593 +106,402 @@ class WebSocketTest {
             webSocket("/echo") {
                 val data = receiveDeserialized<Data>(typeInfo<Data>())
                 sendSerialized(data, typeInfo<Data>())
-                outgoing.send(Frame.Close())
```

Comment:
I've removed explicit sending of `Frame.Close` almost everywhere as it is sent automatically at the end anyway